yarn-project/key-store/src/test_key_store.ts

```diff
@@ -54,9 +54,10 @@ export class TestKeyStore implements KeyStore {
     const publicKeysHash = publicKeys.hash();
     const accountAddress = computeAddress(publicKeysHash, partialAddress);
 
-    // We save the keys to db
+    // We save the keys to db associated with the account address
     await this.#keys.set(`${accountAddress.toString()}-public_keys_hash`, publicKeysHash.toBuffer());
 
+    // Naming of keys is as follows ${from}-${to}_${any modifiers (m = master, h = hash)}
```

Comment:
```suggestion
    // Naming of keys is as follows ${from}-${to}_m
```
The h modifier is no longer used.